server/lib/nl/detection/palm_api.py

```diff
@@ -91,8 +92,11 @@ def parse_response(query: str, resp: Dict, ctr: counters.Counters) -> Dict:
       return {}
 
     try:
-      ans_json = json.loads(ans)
-    except json.decoder.JSONDecodeError as e:
+      # Use json5 to load since it is a lot more lenient (e.g.,
```

Comment:
can put some absolute latency number here? how many seconds for 600x?

Reply:
Done.

Here's my benchmark result: https://gist.github.com/pradh/63e5b9085505a7906062575314d1365a#file-benchmarkjson5-py-L1-L5